docs/pages/eas-insights/introduction.mdx

```diff
@@ -13,6 +17,16 @@ EAS Insights makes it easy to see the state of your app, providing information a
 
 If you're already using [EAS Update](/eas-update/introduction.mdx), we provide certain high-level usage insights without any additional client-side changes. This is possible because by aggregating data from client requests to check for an update into a limited Insights view that shows usage over time, as well as usage broken down by platform.
 
-## Using the `expo-insights` module
+## Use `expo-insights` library
+
+Developers can add the `expo-insights` library to their projects and gain more precise usage metrics (than those provided by just aggregating update requests) and additional breakdowns by app store version. Currently, the library is limited to sending client events only pertaining to cold starts of the app, but in the future we will expand `expo-insights` to offer new types of events and payloads, to support more advanced functionality.
+
+### Installation
+
+> Available for SDK 48 and higher.
+
+To install `expo-insights` in your project, run the following command:
+
+<Terminal cmd={['$ npx expo install expo-insights']} />
 
-Developers can add the `expo-insights` library to their projects and gain more precise usage metrics (than those provided by just aggregating update requests) and additional breakdowns by app store version. Currently, the module is limited to sending client events only pertaining to cold starts of the app, but in the future we will expand `expo-insights` to offer new types of events and payloads, to support more advanced functionality.
+After installing the library, create a new [development build](/develop/development-builds/introduction/).
```

Comment:
```suggestion
After installing the library, [create a new build](/workflow/customizing/).
```

link to be replaced in the future by the docs page that we discussed @amandeepmittal